pallas-applying/src/utils.rs

```diff
@@ -28,17 +28,9 @@ pub use validation::*;
 pub type UTxOs<'b> = HashMap<MultiEraInput<'b>, MultiEraOutput<'b>>;
 
 pub fn get_alonzo_comp_tx_size(mtx: &AlonzoMintedTx) -> u32 {
-    match &mtx.auxiliary_data {
-        Nullable::Some(aux_data) => {
-            (aux_data.raw_cbor().len()
-                + mtx.transaction_body.raw_cbor().len()
-                + mtx.transaction_witness_set.raw_cbor().len()) as u32
-        }
-        _ => {
-            (mtx.transaction_body.raw_cbor().len() + mtx.transaction_witness_set.raw_cbor().len())
-                as u32
-        }
-    }
+    (mtx.auxiliary_data.raw_cbor().len()
+     + mtx.transaction_body.raw_cbor().len()
+     + mtx.transaction_witness_set.raw_cbor().len()) as u32
 }
```

Comment:
Now that we have the `KeepRaw` encapsulating the whole field, this simpler formula should do the work, afaik.